sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze_healthcare_entities_with_cancellation.py

```diff
@@ -58,15 +58,24 @@ def health_with_cancellation(self):
             for revascularization with open heart surgery."
         ]
 
-        poller = text_analytics_client.begin_analyze_healthcare(documents)
-        text_analytics_client.begin_cancel_analyze_healthcare(poller)
-        poller.wait()
+        poller = text_analytics_client.begin_analyze_healthcare_entities(documents)
+        
+        try:
+            cancellation_poller = poller.cancel()
+            cancellation_poller.wait()
+        
+        except Warning as e:
```

Comment:
i think to go with removing our client side validation, you can just remove this part of the sample